src/validate.ts

```diff
@@ -178,6 +178,12 @@ export function response(compiledPath: CompiledPath | undefined,
   if (response === undefined) {
     response = operation.responses['default'];
   }
+  if (response === undefined) {
```

Comment:
Definitely an issue here, thanks for finding it!  Although from my reading of the spec if there's no response defined, no content should be returned.  So maybe there should instead be something at the top of the `validate` function like:
```
 if (schema === undefined) {
  if (!value) {
    return;
  } else {
   return {
      actual: value,
      expected: undefined
    };
  } 
```

Let me know what you think.